src/main/java/com/worldwarofants/game/module/tutorial/TutorialView.java

```diff
@@ -0,0 +1,38 @@
+package com.worldwarofants.game.module.tutorial;
+
+import com.worldwarofants.game.arch.AbstractView;
+import com.worldwarofants.game.arch.console.ConsoleOutputHandler;
+
+public class TutorialView extends AbstractView<TutorialViewModel> {
+
+	private final String START_MESSAGE = "Your colony is under attack! You are on the brink of defeat and you must fall back in order to avoid complete annihilation! ...";
+	private final String FIGHT_MESSAGE = "Your entire colony has been overrun and annihilated; a colony with such a low POPULATION is no match for an enemy of this size!";
+	private final String RETREAT_MESSAGE = "You made the right decision!";
+	
+	public TutorialView(TutorialViewModel viewModel) {
+		super(viewModel);
+	}
+
+	public void renderTutorialScreenStart() {
+		String fightPrompt = String.format("* Fight! - %s ", TutorialCommandHandler.COMMAND_FIGHT);
+		String retreatPrompt = String.format("* Retreat! - %s ", TutorialCommandHandler.COMMAND_RETREAT);
+		String quitPrompt = String.format("* Quit! - %s ", TutorialCommandHandler.COMMAND_QUIT);
```

Comment:
You still have some magical strings left over. These can be called something like COMMAND_FIGHT_DESCRIPTION. The description itself desperately needs to be changed, because still, `fight! - fight` doesn't mean anything. :D

Reply:
I've removed all the magic strings. I also create a few new ConsoleOutputHandler methods to handle  the formatting of standard cases like spelling the title and posting the command description followed by the command string.

As for the command descriptions, I agree in practice they should be descriptive, but right now essentially this is all a placeholder until we add the game logic, so I'm just going to leave it as-is

Reply:
You can add a TODO as a reminder to replace them, if you haven't already.